lib/facter/cisco.rb

```diff
@@ -43,6 +44,10 @@
 
     hash['virtual_service'] = Platform.virtual_services
 
+    hash['feature_compatible_module_iflist'] = {}
+    interface_list = Feature.compatible_interfaces('fabricpath')
+    hash['feature_compatible_module_iflist']['fabricpath'] = interface_list
```

Comment:
I like what you've done with this however I worry about adding this lookup tax to every single facter run. I had the same concern with interface_capabilities (which is only used for beaker) and ultimately decided it was better in that case to keep as an on-demand call.

In this case it looks fairly lightweight and could possibly be used by some of the providers themselves.